snowpack/src/build/import-resolver.ts

```diff
@@ -100,6 +100,16 @@ export function createImportResolver({fileLoc, config}: {fileLoc: string; config
     if (config.packageOptions.external?.includes(spec)) {
       return spec;
     }
+
+    // Note: CSS is allowed to play by different rules. If we can locate a file locally, don’t try and resolve an npm module
+    const isCSSImportingCSS =
+      path.extname(fileLoc) === '.css' &&
+      path.extname(spec) === '.css' &&
+      fs.existsSync(path.join(path.dirname(fileLoc), spec));
+    if (isCSSImportingCSS) {
+      return spec.startsWith('.') ? spec : `./${spec}`;
```

Comment:
In the end, we go ahead and transform it to `./${spec}`. This is still CSS-friendly, and it marks it as a relative import like it should be.